authors/apps/articles/tests/endpoints/test_get.py

```diff
@@ -46,15 +46,17 @@ def test_getArticle_status(self):
     def test_getArticle_content(self):
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
         url = reverse('articles')
-        response = self.client.get(url)
-
-        response.render()
-        self.assertIn(b"life_love_death", response.content)
-        self.assertIn(b"Life Love and Death", response.content)
-        self.assertIn(b"What is life?", response.content)
-        self.assertIn(b"This is the real life body.", response.content)
-        self.assertIn(b"[\"life\",\"love\",\"death\"]", response.content)
-        self.assertIn(b"4", response.content)
+        response = self.client.get(url).render()
+        self.assertIn(b"1", response.content)  # count
+        self.assertIn(b"null", response.content)  # next
+        self.assertIn(b"null", response.content)  # previous
+        self.assertIn(b"1", response.content)  # article id
```

Comment:
There are 2 similar tests achieving different results. I think it would be better to have descriptive test names.